src/client/FlagInputModal.ts

```diff
@@ -30,8 +30,9 @@ export class FlagInputModal extends LitElement {
         >
           ${Countries.filter(
             (country) =>
-              country.name.toLowerCase().includes(this.search.toLowerCase()) ||
-              country.code.toLowerCase().includes(this.search.toLowerCase()),
+              !country.restricted &&
+              (country.name.toLowerCase().includes(this.search.toLowerCase()) ||
```

Comment:
nit: This is a bit confusing
```suggestion
              !(country.restricted ||
                country.name.toLowerCase().includes(this.search.toLowerCase()) ||
```